### ISSUE TITLE
Export as image

### ISSUE THREAD

**Issue by mrouard** [zeleniy/ideogram#17]
A new feature would be to be able to export in different format such as JPEG, PNG and SVG.

Is that possible to use ideogram to generate such pictures via command line? (without setting up an http server)

**Comment by eweitz** [comment]
Ideogram has a command line interface (CLI) made possible via Node.  However, the CLI does not work with newer versions of Ideogram -- since the introduction of support for all eukaryotes.  

See https://github.com/eweitz/ideogram/pull/25 for examples of the Ideogram CLI.  It is [fast](https://github.com/eweitz/ideogram/pull/23), but effectively limited to human and mouse at the moment.  You can get a working version of the CLI from [eweitz:ideogram f56439](https://github.com/eweitz/ideogram/tree/f56439e6e20a3dd40d096879598c1e330afe46d3).

**Comment by mrouard** [comment]
Thanks @eweitz for pointing out to this solution ! I am going to have a look a it.

**Comment by zeleniy** [comment]
@eweitz it doesn't work because of usage [promises](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise)?

**Comment by zeleniy** [comment]
@eweitz yes, problem was partially related with usage of promises. I did not found reason to use at least one of them, remove it and then CLI tool became partially worked. The sake of problem that CLI script stopped before asynchronous promise start to render SVG on page at all.

@mrouard i have wrote new tool based on code of existed tool which doesn't work anyway. Usage:
```
$ cd to/ideogram/directory
$ ls
AUTHORS       README.md     data/         examples/     lib/          package.json  src/          test.log
LICENSE       cli/          dist/         gulpfile.js   node_modules/ scripts/      test/
$ ls cli/
batch-render.js   batch-render2.js  ideo-cli.js*      ideo-cli2.js*
$ node cli/ideo-cli2.js %ideo-config% %bands-data-file% %output-dir%
```
**%ideo-config%** - JSON string. See example [here](https://github.com/zeleniy/ideogram/blob/master/examples/recombination.html#L26-L68).
**%bands-data-file%** - any file from [data/bands/native directory](https://github.com/zeleniy/ideogram/tree/master/data/bands/native). **%ideo-config%** property `organism` must corresponds to data file from that directory.
**%output-dir%** - directory where produced image to store.

But for first time, before run `node` try to install (just once) dependencies which may be not installed yet:
```
npm i
```

Example for human:
```
$ ls images/
ls: images/: No such file or directory
$ node cli/ideo-cli2.js '{"organism":"human"}' data/bands/native/ideogram_9606_GCF_000001305.14_850_V1.js images
$ ls images/
ideogram_9606_GCF_000001305.14_850_V1.png
```
Example for banana:
```
$ ls images/
ls: images/: No such file or directory
$ node cli/ideo-cli2.js '{"organism":"banana","orientation":"horizontal","ploidy":3,"ancestors":{"A":"#dea673","B":"#7396be"},"ploidyDesc":["AAB","AAB","BAB","AAB","AAB","BBB","AAB","AAB","AAB","AAB","AAB"],"rangeSet":[{"chr":1,"ploidy":[0,1,0],"start":17120000,"stop":25120000,"color":[0,"#7396be",0]},{"chr":2,"ploidy":[0,1,1],"start":12120000,"stop":15120000,"color":[0,"#7396be","#dea673"]},{"chr":2,"start":0,"stop":1712000,"color":"red"}]}' data/bands/native/ideogram_banana_v0.1.js images
$ ls images/
ideogram_banana_v0.1.png
```
